core/src/index/knowhere/knowhere/index/Index.h

```diff
@@ -37,11 +37,6 @@ class ToIndexData : public milvus::cache::DataObj {
     explicit ToIndexData(int64_t size) : size_(size) {
     }
 
-    int64_t
-    Size() override {
-        return size_;
-    }
-
```

Comment:
This function is used to provide the current index memory usage. If this function is deleted, LRU may not work properly. Please confirm this deletion can pass all tests.

Reply:
all tests pass